lib/packet.mli

```diff
@@ -285,17 +285,27 @@ type q_class = Q_IN | Q_CS | Q_CH | Q_HS | Q_NONE | Q_ANY_CLS
 val q_class_to_string : q_class -> string
 val string_to_q_class : string -> q_class option
 
-(** mDNS unicast response bit. *)
-type q_unicast = QM | QU
+(** This bit is unused in normal DNS, but in mDNS a value
+    of 0 means that multicast responses are desired,
+    while a value of 1 is a request for unicast responses. *)
```

Comment:
You might also mention that the RFC calls these values `QM` and `QU` and add the line

```
@see <https://tools.ietf.org/html/rfc6762#section-5.4> RFC 6762 Section 5.4
```